raiden/raiden_service.py

```diff
@@ -1255,14 +1255,14 @@ def direct_transfer_async2(self, token_address, amount, target, identifier):
         the target. After the message is sent the target is effectively paid
         and then it is not possible to revert.
 
-        The async result will be set to False iff there is no direct channel
+        The async result will be set to False if there is no direct channel
```

Comment:
https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Ignorant me 😒

Reply:
nah, It's hard to tell when I get words right